manifests/integrations/php_fpm.pp

```diff
@@ -21,12 +21,25 @@
 #
 
 class datadog_agent::integrations::php_fpm(
+  $http_host        = undef,
```

Comment:
No worries, it's such a small change I don't want to bother you with it. I'll take care of it! Thanks!